src/src/rx_main.cpp

```diff
@@ -298,6 +299,9 @@ bool ICACHE_RAM_ATTR HandleSendTelemetryResponse()
         return false; // don't bother sending tlm if disconnected or TLM is off
     }
 
+#if defined(LBT_ACTIVE)
```

Comment:
I would have thought we make the LBT_ACTIVE check a runtime check based on configured power level and dynamic mode enabled.
i.e. 
```cpp
#if defined(Regulatory_Domain_EU_CE_2400)
bool LBTEnabled = config.GetDynamicPower() || config.GetPower() > PWR_10mW;
#else
bool LBEEnabled = false;
#endif
```
This way if a user selects non-dynamic 10mW then it's non-LBT mode and if they select Dynamic then LBT is enabled.

Reply:
yeah, I thought about runtime option as well. However I personally find it a bit appealing to be able to run 10mW mode without any pollution of the runtime from LBT. Maybe that's just me...

The problem with the runtime check as @avsaase implemented in his PR, and it was also pointed out in the other comment, is that care must be taken when to toggle the LBT_active flag, but that can be figured out. 

Let me know what you guys would prefer, and I'll have a look on how to do it?

Reply:
My preference is minimising the number of compile time options.

Reply:
I think it should be a runtime option because I think some pilots will choose to limit to 10mW to avoid skipping packets when there is a lot of interference.

Reply:
seems @avsaase is making progress in his PR based on my fork, I'll follow the discussion there and pull that in

Reply:
I've pulled in your PR @avsaase and I've added safe LBTEnable change functionality in a later commit. From my testing, this seems to work quite nicely, no LBT if configured to 10mW, but LBT enables above 10mW, or with dynamic power, even if dynamic power chooses 10mW.  Feels quite intuitive and should be Good from compliance perspective. Equipment should be allowed to change between Adaptive/non-adaptive runtime like this.